posthog/warehouse/models/datawarehouse_saved_query.py

```diff
@@ -90,6 +104,29 @@ class Meta:
             )
         ]
 
+    @property
+    def query(self) -> dict:
+        if self._query:
+            return self._query
+
+        # Currently these are all revenue analytics queries, so it makes sense to run this without any checks
+        # TODO: Make this more generic to support query types from other sources
+        if self.query_type is not None:
```

Comment:
Why did you decide to do the "creating rev analytics query" here? Could we keep this outside of the saved_query itself? In other words, keep saved_query generic and during the stripe source sync, just have logic that initializes/creates the queries and saves them in as saved queries?